Framework/CurveFitting/src/Algorithms/RefinePowderInstrumentParameters3.cpp

```diff
@@ -1198,6 +1198,8 @@ void RefinePowderInstrumentParameters3::setFunctionParameterFitSetups(const IFun
  * exacly same as
  * source;
  */
+// variable source is moved within this class(so it might be better to be passed by value and not reference)
+// cppcheck-suppress passedByValue
 void duplicateParameters(map<string, Parameter> source, map<string, Parameter> &target) {
```

Comment:
```suggestion
void duplicateParameters(const map<string, Parameter> &source, map<string, Parameter> &target) {
```
I think `source` is not changed in this method, so we can make it `const`